web/libs/ui/src/assets/icons/index.js

```diff
@@ -0,0 +1 @@
+export { ReactComponent as LsCross } from "./cross.svg";
```

Comment:
```suggestion
export { ReactComponent as IconCross } from "./cross.svg";
```

Reply:
Since this is now getting used from the proper place, the name needs to reflect properly what it is. Ls is a namespace based on the application not a particular to the function. Can we update this accordingly here and also let's maybe document this somewhere in the Guild initiatives to call this out.

Reply:
I also considered having LSE using this version of cross.svg - but i noticed that the 2 versions of cross.svg are actually different - which is mildly concerning - I'll have one of the designers chime in on that - would be nice to double down on standardization